snap/snapcraft.yaml

```diff
@@ -53,14 +53,11 @@ passthrough:
       environment:
         LC_ALL: C.UTF-8
         LANG: C.UTF-8
-        PGHOST: $SNAP_COMMON/sockets
         SNAPCRAFT_PRELOAD_REDIRECT_ONLY_SHM: 1
```

Comment:
yes, because they are not only for the kong-postgres part. Without them we get errors such as
```
- Run install hook of "edgexfoundry" snap if present (run hook "install": 
-----
mkdir: cannot create directory '/var/snap/edgexfoundry': Read-only file system
mkdir: cannot create directory '/var/snap/edgexfoundry': Read-only file system
mkdir: cannot create directory '/var/snap/edgexfoundry': Read-only file system
mkdir: cannot create directory '/var/snap/edgexfoundry': Read-only file system
```